tests/openQA/dashboard.pm

```diff
@@ -9,7 +9,11 @@ sub run {
     ensure_unlocked_desktop();
     x11_start_program("firefox http://localhost", 60, { valid => 1 } );
     # starting from git might take a bit longer to get and generated assets
-    assert_screen 'openqa-dashboard', check_var('OPENQA_FROM_GIT', 1) ? 180 : 60;
+    # workaround for poo#19798, basically doubles the timeout
+    if ((check_screen 'openqa-dashboard', 180) == undef) {
```

Comment:
hint for the future: Just take the return value of check_screen as boolean and don't compare for equality with undef, i.e.

```perl
if (!check_screen(…)) …
```